components/PsychrometricCharts.py

```diff
@@ -0,0 +1,177 @@
+import math
+import numpy as np
+import plotly.graph_objs as go
+from dash import Dash, dcc, html
+from dash.dependencies import Input, Output
+
+# 计算函数
+def calculate_saturation_vapor_pressure(temp_c):
+    return 610.94 * np.exp(17.625 * temp_c / (temp_c + 243.04))
+
+def calculate_dew_point(temp_c, relative_humidity):
```

Comment:
this is good but you should use the function from pythermalcomfort so we do not have code repetitions

Reply:
![image](https://github.com/user-attachments/assets/69030323-743d-4869-beb6-e36fac19ad87)
after that i changed the calculation of the parameters of the chart